pkg/machinetool/machinetool.go

```diff
@@ -74,6 +74,11 @@ func Terminate(machine string) error {
 	return err
 }
 
+func ImportRaw(imagePath, imageName string) error {
+	_, err := machinectl(nil, nil, "--verify=no", "import-raw", imagePath, imageName)
```

Comment:
Why is machinectl's [`--verify`](https://www.freedesktop.org/software/systemd/man/machinectl.html) not an option for verification?

Reply:
> Why is machinectl's --verify not an option for verification?

Because systemd only supports special name of GPG key files, which are not supported from CoreOS. AFAIK that was the whole point of writing this code in this place.

Quote from machinectl(1), in the section of pull-tar/pull-raw.

> Verification is done via SHA256SUMS and SHA256SUMS.gpg files that need to be made available on the same web server, under the same URL as the .tar file, but with the same web server, under the same URL as the .tar file, but with the last component (the filename) of the URL replaced. 

Maybe @nhlfr could give us more background.